libs/api-catalogue/elastic/src/lib/queries/search.model.ts

```diff
@@ -82,6 +82,7 @@ export const searchQuery = ({
       { 'title.keyword': { order: 'asc' } },
     ],
     size: limit + 1, // if we have a page number add it as offset for pagination
+    search_after: [] as typeof searchAfter,
```

Comment:
~Please make sure that the ES client does not handle this value differently from the property being skipped previously. I've been 🔥  too many times by similar bugs.

Reply:
I reverted back to the original approach where `search_after` is only added to the query if it contains elements, and added explicit typing for the result so we don't get the linter error: `Unexpected any. Specify a different type.`.